src/composables/fungibleTokens.ts

```diff
@@ -0,0 +1,310 @@
+      };
+    } catch (error) {
+      return {};
+    }
+  }
+
+  async function transfer(
+    address: Encoded.ContractAddress,
+    toAccount: Encoded.AccountAddress,
+    amount: number,
+    option: {
+      waitMined: boolean,
+      modal: boolean,
+    },
+  ) {
+    const aeSdk = await getAeSdk();
+    const tokenContract = await aeSdk.initializeContract({
+      aci: FungibleTokenFullInterfaceACI,
+      address,
+    });
+    return tokenContract.transfer(toAccount, amount.toFixed(), option);
+  }
+
+  async function burnTriggerPoS(
+    address: Encoded.ContractAddress,
+    posAddress: string,
+    invoiceId: string,
+    amount: number,
+    option: {
+      waitMined: boolean,
+      modal: boolean,
+    },
+  ) {
+    const aeSdk = await getAeSdk();
+    const tokenContract = await aeSdk.initializeContract({
+      aci: ZeitTokenACI,
+      address,
+    });
+    return tokenContract.burn_trigger_pos(
+      amount.toFixed(),
+      posAddress,
+      invoiceId,
+      option,
+    );
+  }
+
+  function getTxSymbol(transaction?: ITransaction) {
+    if (transaction?.pendingTokenTx) {
+      return availableTokens.value[transaction.tx.contractId]?.symbol;
+    }
+    const contractCallData = transaction?.tx && categorizeContractCallTxObject(transaction);
+    return availableTokens.value[contractCallData?.token!]?.symbol || AE_SYMBOL;
+  }
+
+  function getTxAmountTotal(
+    transaction: ITransaction,
+    direction: string = TX_DIRECTION.sent,
+  ) {
+    const contractCallData = transaction.tx && categorizeContractCallTxObject(transaction);
+    if (contractCallData && availableTokens.value[contractCallData.token!]) {
+      return +toShiftedBigNumber(
+        contractCallData.amount || 0,
+        -availableTokens.value[contractCallData.token!].decimals,
+      );
+    }
+    const isReceived = direction === TX_DIRECTION.received;
+
+    const rawAmount = (
+      transaction.tx?.amount
+      || (transaction.tx?.tx?.tx as any)?.amount
+      || transaction.tx?.nameFee
+      || 0
+    );
+
+    const amount: BigNumberPublic = (typeof rawAmount === 'object')
+      ? rawAmount
+      : new BigNumber(Number(rawAmount));
+
+    return +aettosToAe(amount
+      .plus(isReceived ? 0 : transaction.tx?.fee || 0)
+      .plus(isReceived ? 0 : transaction.tx?.tx?.tx?.fee || 0));
+  }
+
+  onNetworkChange(async (network, oldNetwork) => {
+    const newMiddlewareUrl = network.protocols[PROTOCOL_AETERNITY].middlewareUrl;
+    const oldMiddlewareUrl = oldNetwork?.protocols?.[PROTOCOL_AETERNITY]?.middlewareUrl;
+    if (newMiddlewareUrl !== oldMiddlewareUrl) {
+      await loadAvailableTokens();
+      await loadTokenBalances();
+    }
+  });
+
+  watch(aeAccounts, (val, oldVal) => {
+    if (val !== oldVal) {
+      loadTokenBalances();
+    }
+  });
+
+  (() => {
+    if (!isInitialized) {
+      availableTokensWatcher(() => loadAvailableTokens());
+      tokenBalancesWatcher(() => loadTokenBalances());
+    }
+  })();
```

Comment:
```suggestion
  availableTokensWatcher(() => loadAvailableTokens());
  tokenBalancesWatcher(() => loadTokenBalances());
```
Polling watcher should be fired every time the composable is used. This allows it to count how many components are using this composable.